customization/dark-theme.md

```diff
@@ -120,7 +120,7 @@ The advantage of creating a dark themed Template VM is, that each AppVM which is
 
 1. Start VM
 
-    **Note:** In case of App VM start the Template on which the AppVM is based on.
+    **Note:** In the case of App VM start the Template on which the AppVM is based on.
```

Comment:
Yeah, I meant the whole sentence sounded weird, not your change in particular. (same with most of my nitpicks here)

"If you want to persistently affect an AppVM, remember you'll need to make the changes to its TemplateVM instead, so start that here." ?